packages/core-ethereum/src/indexer/reducers.fixtures.spec.ts

```diff
@@ -1,190 +1,230 @@
-  closureByPartyA: false
-})
-
-export const FUNDED_CHANNEL_2 = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(10),
-  partyABalance: new BN(3),
-  closureTime: new BN(0),
-  stateCounter: new BN(1),
-  closureByPartyA: false
-})
-
-export const OPENED_CHANNEL = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(10),
-  partyABalance: new BN(3),
-  closureTime: new BN(0),
-  stateCounter: new BN(2),
-  closureByPartyA: false
-})
-
-export const REDEEMED_CHANNEL = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(10),
-  partyABalance: new BN(4),
-  closureTime: new BN(0),
-  stateCounter: new BN(2),
-  closureByPartyA: false
-})
-
-export const CLOSING_CHANNEL = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(10),
-  partyABalance: new BN(4),
-  closureTime: new BN(1611671775),
-  stateCounter: new BN(3),
-  closureByPartyA: true
-})
-
-export const REDEEMED_CHANNEL_2 = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(10),
-  partyABalance: new BN(2),
-  closureTime: new BN(1611671775),
-  stateCounter: new BN(3),
-  closureByPartyA: false
-})
-
-export const CLOSED_CHANNEL = new ChannelEntry(undefined, {
-  blockNumber: new BN(0),
-  transactionIndex: new BN(0),
-  logIndex: new BN(0),
-  deposit: new BN(0),
-  partyABalance: new BN(0),
-  closureTime: new BN(0),
-  stateCounter: new BN(10),
-  closureByPartyA: false
-})
+} as Event<'ChannelClosed'>
+
+export const EMPTY_ACCOUNT = new AccountEntry(new Address(new Uint8Array()))
+
+export const INITIALIZED_ACCOUNT = new AccountEntry(partyA, partyAPubKey, secret1, new BN(1))
+
+export const SECRET_UPDATED_ACCOUNT = new AccountEntry(partyA, partyAPubKey, secret2, new BN(2))
+
+export const EMPTY_CHANNEL = new ChannelEntry(
+  parties,
+  new BN(0),
+  new BN(0),
+  new BN(0),
+  new BN(0),
+  false,
+  new BN(0),
+  new BN(0)
+)
+
+export const FUNDED_CHANNEL = new ChannelEntry(
+  parties,
+  new BN(3),
+  new BN(3),
+  new BN(0),
+  new BN(0),
+  false,
+  new BN(0),
+  new BN(0)
+)
+
+export const FUNDED_CHANNEL_2 = new ChannelEntry(
+  parties,
+  new BN(10),
+  new BN(3),
+  new BN(0),
+  new BN(0),
+  false,
+  new BN(0),
+  new BN(0)
+)
+
+export const OPENED_CHANNEL = new ChannelEntry(
+  parties,
+  new BN(10),
+  new BN(3),
+  new BN(0),
+  new BN(1),
+  false,
+  new BN(0),
+  new BN(0)
+)
+
+export const REDEEMED_CHANNEL = new ChannelEntry(
+  parties,
+  new BN(10),
+  new BN(4),
+  new BN(0),
+  new BN(1),
+  false,
+  new BN(0),
+  new BN(0)
+)
+
+export const CLOSING_CHANNEL = new ChannelEntry(
+  parties,
+  new BN(10),
+  new BN(4),
+  new BN(1611671775),
```

Comment:
I know these are just tests but could we maybe constant' these numbers? I hate magic numbers in general.

Reply:
agreed, in a later PR am planning to implement `ChannelEntry.fromObject`